polars/polars-io/Cargo.toml

```diff
@@ -13,22 +13,19 @@ repository = "https://github.com/ritchie46/polars"
 # support for arrows json parsing
 json = []
 # support for arrows ipc file parsing
-ipc = ["arrow/ipc"]
+ipc = ["arrow/io_ipc"]
 #ipc = []
 lazy = []
-parquet = ["polars-core/parquet", "parquet_lib"]
+parquet = ["polars-core/parquet"]
 dtype-u64 = ["polars-core/dtype-u64"]
 dtype-date64 = ["polars-core/dtype-date64", "polars-core/temporal"]
 dtype-date32 = ["polars-core/dtype-date32"]
-csv-file = ["csv", "csv-core", "memmap", "fast-float", "lexical", "arrow/csv"]
+csv-file = ["csv", "csv-core", "memmap", "fast-float", "lexical", "arrow/io_csv"]
 #csv-file = ["csv", "csv-core", "memmap", "fast-float", "lexical"]
 
 [dependencies]
-arrow = {git = "https://github.com/apache/arrow-rs", rev = "0f55b828883b3b3afda43ae404b130d374e6f1a1", default-features=false}
+arrow = { package = "arrow2", git = "https://github.com/jorgecarleitao/arrow2", rev = "a0068a8ae892b281f7c66f0d1ee7396cb20f753a" }
 #arrow = {version = "4.2", default-features = false }
-parquet_lib = {optional = true, package="parquet", git = "https://github.com/apache/arrow-rs", rev = "0f55b828883b3b3afda43ae404b130d374e6f1a1"}
-#parquet_lib = {version = "4.2", package="parquet", optional = true}
-polars-arrow = {version = "0.14.1", path = "../polars-arrow"}
```

Comment:
arrow2 dependency with parquet is a bit different:

* parquet2 <- arrow2
* arrow2 <- polars

parquet is under a feature gate `io_parquet`.